### PR TITLE
Danish translation 

### PR DESCRIPTION
**Pull request by cryptobench**

### CODE REVIEW

File: i18n/da/building.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+
+### Andre muligheder du kan bruge i stedet for trin 2:
+
+For en akkurat og komplet liste af muligheder, se targets i [Makefile](https://github.com/monero-project/kovri/blob/master/Makefile).
+
+#### Noter
+- Alle bygge resultater (inkluderet Doxygen) ville være i bygge resultatets mappe
+
+### Clang
+For at bygge med clang, **skal* du eksportere følgende:
```
Comment (anonimal) [inline i18n/da/building.md]: >`**skal*`

You missed ending the word with another `*`. The remaining text is now bold unless you close the word with another `*`.

Comment (cryptobench) [reply i18n/da/building.md]: Will fix in a sec :)

**Comment by anonimal** [comment]
One that issue is fixed, can you please rebase all these commits into a single signed commit?

**Comment by cryptobench** [comment]
I will have to watch a guide for how to turn all of the commits into a single signed then. I'm pretty new to Github :p gimme some time

**Comment by cryptobench** [comment]
Is this the right thing? https://help.github.com/articles/signing-commits-using-gpg/

**Comment by erciccione** [comment]
@cryptobench no, check out this https://davidwalsh.name/squash-commits-git
If you need further help, see you on #monero-translations

**Comment by cryptobench** [comment]
Can we delete this pull request? I did a new one which isnt messy because of all the commits.

**Comment by erciccione** [comment]
New PR is #36

**Pull request closed without merging**